terraform/development/main.tf

```diff
@@ -80,13 +81,46 @@ module "application" {
   ecs_security_group_id                = module.application_security_group.ecs_security_group_id
   export_bucket_access_policy_arn      = module.cds_export.read_write_policy_arn
   export_bucket_details                = module.cds_export.details
+  govuk_notify_api_key_secret_arn      = module.application_secrets.govuk_notify_api_key_secret_arn
   load_balancer_target_group_arn       = module.front_door.load_balancer_target_group_arn
+  os_data_key_secret_arn               = module.application_secrets.os_data_key_secret_arn
   private_subnet_ids                   = module.networking.private_subnet_ids
   rails_env                            = local.rails_env
+  rails_master_key_secret_arn          = module.application_secrets.rails_master_key_secret_arn
   redis_connection_string              = module.redis.redis_connection_string
+  sentry_dsn_secret_arn                = module.application_secrets.sentry_dsn_secret_arn
   sns_topic_arn                        = module.monitoring.sns_topic_arn
 }
 
+moved {

```

Comment:
These shouldn't be needed for dev, although they shouldn't cause errors by being here.

Reply:
Good point! I'll remove to save any confusion